### PR TITLE
NTR: Add Media Upload Helper

### PR DESCRIPTION
**Pull request by boxblinkracer**
proposal pull request, please check if you like it

building a media upload helper to automatically insert local assets from the fixture folder as cover images or other media

```php
$this->helper->Media()->upload(
    self::MEDIA_ID,
    $this->helper->getMediaDefaultFolderId('product'),
     __DIR__ . '/Assets/voucher-product.png',
    'png',
    'image/png',
);
```

Screen Shot 2022-07-27 at 23 48 02

### CODE REVIEW

**Comment by jkniest** [comment]
Could you please run php cs fixer? Than the diff is smaller to review 😝

**Comment by boxblinkracer** [comment]
just wanted to have a first feedback on the idea @jkniest 
code is not yet done....but can we add such a thing? what do you think about the concept (please see PR description with code example)

**Comment by jdambacher** [comment]
I like the feature!

File: src/Utils/MediaUtils.php [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+use Shopware\Core\Framework\DataAbstractionLayer\EntityRepositoryInterface;
+
+class MediaUtils
+{
+    private EntityRepositoryInterface $mediaRepository;
+
+    /**
+     * @var FileSaver
+     */
+    private $fileSaver;
```
Comment (jdambacher) [inline src/Utils/MediaUtils.php:19]:
```suggestion
    private FileSaver $fileSaver;
```

File: src/Utils/MediaUtils.php [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+    /**
+     * @var FileFetcher
+     */
+    private $fileFetcher;
```
Comment (jdambacher) [inline src/Utils/MediaUtils.php:24]:
```suggestion
    private FileFetcher $fileFetcher;
```

File: src/Utils/MediaUtils.php [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+class MediaUtils
+{
+    private EntityRepositoryInterface $mediaRepository;
+
+    /**
+     * @var FileSaver
+     */
+    private $fileSaver;
+
```
Comment (jdambacher) [inline src/Utils/MediaUtils.php:20]:
```suggestion
```

File: src/Utils/MediaUtils.php [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+use Shopware\Core\Content\Media\File\FileFetcher;
+use Shopware\Core\Content\Media\File\FileSaver;
+use Shopware\Core\Framework\Context;
+use Shopware\Core\Framework\DataAbstractionLayer\EntityRepositoryInterface;
+
+class MediaUtils
+{
+    private EntityRepositoryInterface $mediaRepository;
+
```
Comment (jdambacher) [inline src/Utils/MediaUtils.php:15]:
```suggestion
```

**Review by jdambacher — CHANGES_REQUESTED** [review]

**Pull request merged**